doc/workflow/coding_standard_qml.md

```diff
@@ -0,0 +1,175 @@
+# Coding Standard QML
+
+###1. QML document structure
+
+#####Rule 1.1. Basic document and elements structure should be order like this:
```

Comment:
I'd like to have some codes for the rules. Normally, it is very hard to distinguish between rules from diferrent sectionsa and very hard to remeber (though it is hard to remember every rule from coding standard anyway). My suggestion is to assign every rule the code that consists from shortened word and rule number. Example (for QML structure section): Struct1, Struct2, (for code formatting section): Format1, Format2, ...